core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/operators/OneToOneOpExecConfig.scala

```diff
@@ -25,7 +25,7 @@ class OneToOneOpExecConfig(override val tag: OperatorIdentifier, val opExec: Int
           LayerTag(tag, "main"),
           opExec,
           Constants.defaultNumWorkers,
-          FollowPrevious(),
+          UseAll(),
```

Comment:
I think FollowPrevious() is the correct DeploymentFilter for OneToOneOpExecConfig because otherwise it is no more One-To-One. But I understand the problem you are highlighting here. This issue originates because we have given the worker placement control only to an operator. But the mysql-source case shows that while placing workers we also need to look at the previous operator. @shengquan-ni  comments?

Reply:
followPrevious only means the machines this operator uses will be the same as the previous operator. numWorker decides how many worker actors should be created. So it's ok to keep followPrevious there.